config/mobility.yml

```diff
@@ -23,9 +23,14 @@ pages:
   - Turn-by-Turn:
     - 'Overview': 'turn-by-turn/overview.md'
     - 'API reference': 'turn-by-turn/api-reference.md'
-    - 'Tutorial': 'turn-by-turn/add-routing-to-a-map.md' 
+    - 'Tutorial': 'turn-by-turn/add-routing-to-a-map.md'
   - 'Optimized route': 'optimized/api-reference.md'
   - 'Time-Distance Matrix': 'matrix/api-reference.md'
+  - 'Isochrones': 'isochrones/api-reference.md'
```

Comment:
@rmglennon I'd suggest changing the TOC listing to read "Isochrone" singular, so it's in sync with other mentions of the product name.